packages/xo-server-backup-reports/src/index.js

```diff
@@ -145,42 +259,163 @@ class BackupReportsXoPlugin {
     this._xo.on('job:terminated', this._report)
   }
 
-  test({ runId }) {
-    return this._backupNgListener(undefined, undefined, undefined, runId)
+  async test({ runId }) {
+    return this._wrapper(undefined, undefined, undefined, runId, true)
   }
 
   unload() {
     this._xo.removeListener('job:terminated', this._report)
   }
 
-  _wrapper(status, job, schedule, runJobId) {
-    if (job.type === 'metadataBackup') {
-      return
+  async _wrapper(status, job, schedule, runJobId, test) {
```

Comment:
Why this new boolean param?

Reply:
To identify the case of testing the plugin. But it can be replaced in the below condition by `job === undefined`.

Reply:
Can't you find a way to make it work with only the logs, so that all cases are handled the same way?